package.xml

```diff
@@ -1,28 +1,22 @@
 <?xml version="1.0"?>
-<package format="2">
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
   <name>mqtt_bridge</name>
-  <version>0.2.1</version>
+  <version>0.1.1</version>
```

Comment:
version nr should increase, e.g. 0.3.0

Reply:
For version number, since it is first version for ROS2 I believe it should be kept as 0.1.1

Reply:
I prefer to increase the number to `0.3.0` to easily distinguish between ROS1 and ROS2.